src/Core/MailTemplates/Handlebars/InvoiceUpcoming.text.hbs

```diff
@@ -9,7 +9,7 @@ Summary Of Charges
 {{/each}}
 {{/if}}
 
-To avoid any interruption in service, please ensure that your payment method on file is up to date and can be charged for the above amount. You can manage your subscription, payment method, and invoices by logging into the web vault at {{{WebVaultUrl}}}.
+To avoid any interruption in service, please ensure that your payment method on file is up to date and can be charged for the above amount. You can manage your subscription, payment method, and invoices by logging into the web vault. {{{UpdateBillingInfoUrl}}}
```

Comment:
For the plain text version, I suggest keeping it in a sentence so there's context for what the link is. For example:

```suggestion
To avoid any interruption in service, please ensure that your payment method on file is up to date and can be charged for the above amount. You can manage your subscription, payment method, and invoices by logging into the web vault. For more information, please visit {{{UpdateBillingInfoUrl}}}.
```

Please ask Priya or Sukhleen for wording.

Reply:
I confirm from Mica. It is a really good suggestion. Thanks